aspnetcore/mvc/views/tag-helpers/built-in/ImageTagHelper.md

```diff
@@ -1,4 +1,18 @@
-﻿# ImageTagHelper
+﻿title: Anchor Tag Helper | Microsoft Docs
+author: Peter Kellner
+description: Shows how to work with Image Tag Helper
```

Comment:
This will become the Twitter description and the meta description, so imo it should be a complete sentence with punctuation.